bin/weewx/drivers/te923.py

```diff
@@ -562,7 +566,7 @@ class TE923Configurator(weewx.drivers.AbstractConfigurator):
     }
     
     city_dict = {
-        0: ["ADD", 3, 0, 9, 0o1, "N", 38, 44, "E", "Addis Ababa, Ethiopia"],
+        0: ["ADD", 3, 0, 9, 1, "N", 38, 44, "E", "Addis Ababa, Ethiopia"],
```

Comment:
This is unrelated cleanup of automated 01 -> 0o1 octal change, better stated as simply 1.  Zero functionality change here.